galaxy.yml

```diff
@@ -0,0 +1,62 @@
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: confluent
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: kafka
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 6.2.0
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+  - Confluent Ansible Community (https://github.com/confluentinc/cp-ansible)
+
+### OPTIONAL but strongly recommended
+# A short summary description of the collection
+description: Confluent Platform Ansible Collection
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+license:
+  - Apache-2.0
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
+license_file: LICENSE.md
```

Comment:
Use `license` or `license_file`?

> mutually exclusive with 'license'

Reply:
license_file to match https://github.com/ansible-collections/community.general/blob/main/galaxy.yml#L8

Reply:
this looks good